config/initializers/paper_trail.rb

```diff
@@ -1 +1,7 @@
 PaperTrail.serializer = PaperTrail::Serializers::JSON
+
+Rails.application.configure do
+  console do
+    PaperTrail.request.whodunnit = "Rails Console"
```

Comment:
In `ECF` the rails console is asking person for a name, I think it was a good practice.

Reply:
Yeah, good call - I'd debated that approach, I'll add that in :+1: